tests/01http-server.pl

```diff
@@ -128,11 +128,17 @@ package SyTest::HTTPServer {
 
       my $method = $request->method;
       my $path = uri_unescape $request->path;
+      my $qs = $request->query_string;
+      if ( defined $qs ) {
+         $qs = "?" . $qs;
+      } else {
```

Comment:
Don't cuddle the `else`; it wants to be free. On its own line.

    }
    else {